infrastructure/Dockerfile

```diff
@@ -1,10 +1,20 @@
 FROM openjdk:8-jdk-alpine3.8
 
 ARG CORFU_JAR
+ARG CORFU_TOOLS_JAR
+
 WORKDIR /app
 
 RUN apk add --update iptables sudo
 
+COPY target/${CORFU_JAR} /usr/share/corfu/lib/${CORFU_JAR}
+COPY target/${CORFU_TOOLS_JAR} /usr/share/corfu/lib/${CORFU_TOOLS_JAR}
+
+COPY target/bin /usr/local
```

Comment:
shouldn't this be `/user/share/corfu/bin` ?

Reply:
fixed